templates/workspace_services/azureml/terraform/get_nsg_priorities.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
 
-set -e
 
 eval "$(jq -r '@sh "nsg_name=\(.nsg_name) resource_group_name=\(.resource_group_name) nsg_rule_name=\(.nsg_rule_name) direction=\(.direction)"')"
-if NSG_RULE=$(az network nsg rule show -g $resource_group_name --nsg-name $nsg_name --name $nsg_rule_name -o json); then
-    NSG_RULE_PRIORITY=$(echo $NSG_RULE | jq '.priority')
+
+# This variables are loaded in for us
```

Comment:
nit: Same suggest regarding THESE rather than THIS

Reply:
done